Auth0/OAuth2Grant.swift

```diff
@@ -30,11 +31,21 @@ protocol OAuth2Grant {
 
 struct ImplicitGrant: OAuth2Grant {
 
+    let authentication: Authentication
     let defaults: [String: String]
     let responseType: [ResponseType]
-
-    init(responseType: [ResponseType] = [.token], nonce: String? = nil) {
+    let leeway: Int
```

Comment:
Shouldn't this be optional as well, with a default set?

Reply:
If it has a default value, then it shouldn't be optional (because it will never be nil).